src/v2/ion/IonResponseHelper.js

```diff
@@ -158,8 +158,14 @@ const isIdxSessionExpiredError = (response) => {
   return errorI18NKey && errorI18NKey === 'idx.session.expired';
 };
 
+const isIdxTerminalResponse = (response) => {
+  const { neededToProceed, interactionCode } = response;
```

Comment:
you can check resp.status directly, as it's evaluated by same logic in auth-js (also to avoid same code).
or use status directly in FormController

Reply:
okay cool, change has been pushed